google/api_core/exceptions.py

```diff
@@ -145,6 +148,17 @@ def errors(self):
         """
         return list(self._errors)
 
+    @property
+    def error_info(self):
+        """Information contained in google.rpc.error_details.ErrorInfo
+
+        Reference:
+            https://github.com/googleapis/googleapis/blob/master/google/rpc/error_details.proto#L112
```

Comment:
To appease Sphinx, please add a blank line here (see the Kokoro failure).  Run `nox -s docs` to verify the fix.

Reply:
Done